packages/stokhos/src/sacado/kokkos/pce/tpetra/Stokhos_Tpetra_ETI_Helpers_UQ_PCE_DEVICE.hpp

```diff
@@ -46,23 +46,43 @@
 #define INSTANTIATE_UQ_PCE_STORAGE(INSTMACRO, STORAGE, LO, GO, N)      \
   INSTMACRO( Sacado::UQ::PCE<STORAGE>, LO, GO, N )
 
+#define INSTANTIATE_UQ_PCE_STORAGE_SD(INSTMACRO, STORAGE, N)      \
```

Comment:
"SD" is the right name -- Node has all the typedefs that `Kokkos::Device` has, so it's harmless use to Node where a `Kokkos::Device` would do.  We will _eventually_ get rid of Node ;-) .